development/tools/jobs/kyma/tests_test.go

```diff
@@ -89,6 +89,7 @@ var tests = []struct {
 	{path: "integration/dex", image: tester.ImageGolangBuildpack1_16, suite: tester.NewGenericComponentSuite,
 		additionalOptions: []jobsuite.Option{
 			jobsuite.JobFileSuffix("tests-generic"),
+			jobsuite.Until(releases.Release124),
```

Comment:
You included this `jobsuite.Until()` command for path `integration/dex`. What about component tests in `dex-static-user-configurer"`? You included `release_until: "1.24"` annotation - isn't it necessary to also add it here to some jobsuite or it is removed and not needed?

Reply:
good catch. thx